runtime/parser/expression_test.go

```diff
@@ -5889,6 +5889,18 @@ func TestParseCasting(t *testing.T) {
 	)
 }
 
+func testParseIdentifiersWith(t *testing.T, identifiers []string, condition func(*testing.T, string, error)) {
+	for _, name := range identifiers {
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
This use of t.Parallel is broken, this will have non-deterministic behaviour (`name` will be bound to some arbitrary value in the test case)